runscope/resource_runscope_step_test.go

```diff
@@ -201,6 +201,7 @@ resource "runscope_step" "main_page" {
   bucket_id      = "${runscope_bucket.bucket.id}"
   test_id        = "${runscope_test.test.id}"
   step_type      = "request"
+  note           = "Testing step, single step test"
```

Comment:
Can you test the value is being saved correcting in the [testAccCheckStepMainPageExists](https://github.com/terraform-providers/terraform-provider-runscope/blob/4841756551904a0e0609f276a3948c81e60fbf1f/runscope/resource_runscope_step_test.go#L158) func?

Reply:
Yeah, I seem to have forgotten a bit of code I intended to add there, will fix ASAP.